README.md

```diff
@@ -107,6 +107,20 @@ EOF
 systemctl enable scanpi --now
 ```
 
+## Docker Compose Template
+```
```

Comment:
Add a devices section to the template and fill it with a basic placeholder value. To share a USB device with a docker container it needs to be mapped explicitly.

https://www.homeautomationguy.io/blog/docker-tips/accessing-usb-devices-from-docker-containers